src/features/home/components/modules/HighlightOfferModule.tsx

```diff
@@ -76,73 +81,88 @@ const UnmemoizedHighlightOfferModule = (props: HighlightOfferModuleProps) => {
 
   return (
     <Container>
-      <StyledTitleContainer>
-        <StyledTitle>{props.highlightTitle}</StyledTitle>
-      </StyledTitleContainer>
-      <Spacer.Column numberOfSpaces={5} />
-      <View>
-        <ColorCategoryBackground
-          start={{ x: 0, y: 0 }}
-          end={{ x: 0, y: 1 }}
-          colors={gradientColorsMapping[props.color]}
-          height={
-            isDesktopViewport
-              ? DESKTOP_COLOR_BACKGROUND_HEIGHT
-              : getColorBackgroundHeight(offerDetailsHeight)
-          }
+      {isNewExclusivityModule ? (
+        <MarketingBlockExclusivity
+          offerId={Number(offerId)}
```

Comment:
Pour un meilleur typage (`offerId` peut être `undefined`) et une cohérence avec le reste du fichier (cf.  `FavoriteButton`) :
```suggestion
          offerId={parseInt(highlightOfferId)}
```